src/actions/actionCreators.js

```diff
@@ -76,6 +76,12 @@ import {
   SET_SHOW_EMPTY_CLAIM_CATEGORIES,
   ADD_NEW_CATEGORY,
   SET_NEW_CATEGORY,
+  SET_CLAIM_VISIBILITY,
+  SET_SHOW_HIDDEN_CLAIMS,
+  UPDATE_SELECTED_CLAIMS,
+  CLEAR_SELECTED_CLAIMS,
+  UPDATE_CATEGORY_FOR_CLAIMS,
+  MOVE_CLAIMS_TO_CATEGORY,
```

Comment:
Why do we keep all action creators in one file. We can split this file up based on domain, for example claims, categories, identity etc.
You can even maintain the import path just by creating an actionCreators directory and moving the exports into the index.js of it.